$MirOS: ports/mailnews/pine/patches/patch-imap_src_imapd_Makefile,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
--- imap/src/imapd/Makefile.orig	Tue Jun 29 21:26:06 2004
+++ imap/src/imapd/Makefile	Wed May  4 23:28:07 2005
@@ -45,7 +45,7 @@ SHELL= /bin/sh
 # Get local definitions from c-client directory
 
 C = ../c-client
-CCLIENTLIB = $C/c-client.a
+CCLIENTLIB = -L$C -lc-client
 CC = `cat $C/CCTYPE`
 CFLAGS = -I$C `cat $C/CFLAGS` $(NSBD) $(ENBD) -DANOFILE=\"$(ANO)\" \
 	-DALERTFILE=\"$(ALERT)\" -DNNTPFILE=\"$(NNTP)\" \
